src/test/java/com/notably/logic/commands/suggestion/DeleteSuggestionCommandTest.java

```diff
@@ -128,4 +136,49 @@ public void execute_generateResponseCorrectly() {
             assertEquals(expectedInput, input);
         }
     }
+
+    @Test
+    public void execute_validPathWithoutPrefix_generateResponseCorrectly() {
+        model.setInput("delete /CS2103");
```

Comment:
Let's make this `COMMAND_WORD + " " + toCs2103.getStringRepresentation()` for consistency?